roles/azimuth/defaults/main.yml

```diff
@@ -265,6 +265,7 @@ azimuth_release_defaults:
       verifySsl: "{{ azimuth_openstack_verify_ssl }}"
       createInternalNet: "{{ azimuth_openstack_create_internal_net }}"
       internalNetCidr: "{{ azimuth_openstack_internal_net_cidr }}"
+      externalNetTemplate: "{{ azimuth_openstack_external_net_name | default(omit) }}"
```

Comment:
`omit` only works to optionally specify a module parameter, not as a dictionary key (it actually produces some weird placeholder string that Ansible knows to interpret as a module argument).

You'll need to turn this into something more like what happens for the theme, using `combine`.